src/components/ui/Notification.tsx

```diff
@@ -26,21 +26,21 @@ interface NotificationProps {
   title?: string;
 }
 
-export const Notification = ({
+export const MyNotification = ({
```

Comment:
The same here with `My` prefix

Reply:
This could be removed

Reply:
I think `Notice` will work, if not, you can add it to exceptions.

Reply:
And don't forget to rename file (for Navigation too)